src/client/sandbox/node/window.js

```diff
@@ -29,6 +31,9 @@ const arrayConcat = Array.prototype.concat;
 
 const HTTP_PROTOCOL_RE = /^http/i;
 
+const ALLOWED_SERVICE_WORKER_PROTOCOLS  = ['https:', 'wss:', 'file:', 'chrome-extension:'];
```

Comment:
`chrome-extention` is not necessary. Url with `chrome-extention` protocol will not modified by `getProxyUrl` function and browser will perform secure origin validation itself.